README.md

```diff
@@ -34,9 +36,41 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Quick Installation via Helm
+## Installation via Helm
+
+### Quick Installation
 
 StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then simply run the helm quick install script (or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options)
```

Comment:
- Remove word `simply` - it may be annoying for users who struggle with this step (for any reason).
- There is no need for parenthesis around `or proceed...` - remove it. Remember that in technical writing, parentheses can always be ignored. Does this sentence convey the same message If we remove all the text between them?

Reply:
Good to know, I'll remember it. On that note, do we have a style guide for technical writing anywhere?

Reply:
This may be too much to gather in a guide and I am not aware of any guide existing in Confluence/Rover. May be there is something at the level of RedHat? Maybe a course?

I learned from this book https://www.goodreads.com/en/book/show/601222.Bugs_in_Writing I can truly recommend it, but it may be difficult to get.